src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -431,7 +437,36 @@ def parse_actions_from_repositories(allow_or_remove_repository):
 
     return actions
 
+def prepare_source_location(source_location, client_registries, registry_name, resource_group_name):
+    if source_location is None or source_location.lower() == ACR_NULL_CONTEXT:
```

Comment:
I know you simply moved this function around, but here is a comment:

What if the user passes an empty string? What behavior do we want? Should it also be handled as the null context or crash below when we check for remote repo? If you want `""` to resolve to `ACR_NULL_CONTEXT`, then consider:

```suggestion
    if not source_location or source_location.lower() == ACR_NULL_CONTEXT:
```